src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -117,17 +128,20 @@ class ServerListWindow final : public Window
     {
         _serverList = {};
         _fetchFuture = {};
+        CheckTextBoxFocus();
```

Comment:
You could just call `ConfigSaveDefault();` here instead.

Reply:
Feels silly to not have seen that, I was too focused on the concept of losing focus to see the clearer solution: [b147c0d](https://github.com/OpenRCT2/OpenRCT2/pull/20004/commits/b147c0dfb23ff4a1af469355ba37d0e69c8fdc8c)